SectionGrid.js

```diff
@@ -129,15 +143,7 @@ class SectionGrid extends Component {
     return (
       <SectionList
         sections={groupedSections}
-        renderItem={({ item, index, section }) => this.renderRow({
-          rowItems: item,
-          rowIndex: index,
-          section,
-          isFirstRow: index === 0,
-          itemsPerRow,
-          rowStyle,
-          containerStyle,
-        })}
+        renderItem={wrapRenderItem(this.props.renderItem)}
```

Comment:
Is this required? Since the renderItem is already present in section, it can be directly accessed from `renderRow` right? Something like this inside `renderRow` function

`const renderItem = this.props.renderItem || section.renderItem`

and then use the newly defined renderItem method in line 75.

Reply:
Than you for your comment. Yes, you are right, that should work :smile: I will update the PR.

Reply:
No problem 👍 Please update when you have time and I will merge it.